include/APIClient/Router.hpp

```diff
@@ -5,24 +5,27 @@
 
 using PutFormData = std::map<std::string, std::string>;
```

Comment:
Is it useful for the user to have? Or is it just use internally?
You can probably put it in the namespace `APIClient` or in the class `Router` if it's useful for the user of the class.